src/OpenAL/Silk.NET.OpenAL/OpenALLibraryNameContainer.cs

```diff
@@ -11,10 +11,10 @@ namespace Silk.NET.OpenAL
     internal class OpenALLibraryNameContainer : SearchPathContainer
```

Comment:
we very likely want to support either OpenAL Soft or OpenAL native here. Will look at how this could be done.

Reply:
This needs more architectural decisions. Temporarily:
- make a new internal class called OpenALSoftLibraryNameContainer
- add an optional parameter to AL.GetApi (i.e. `AL.GetApi(bool soft = false)`)
- use OpenALSoftLibraryNameContainer if soft is true, OpenALLibraryNameContainer otherwise
- this new overload will probably change and be deprecated after the first preview of 3.0